workflows/mettannotator.nf

```diff
@@ -412,11 +421,13 @@ workflow METTANNOTATOR {
         PROKKA.out.gff.join(
             INTERPROSCAN.out.ips_annotations
         ).join(
-           EGGNOG_MAPPER_ANNOTATIONS.out.annotations, remainder: true
```

Comment:
I think there was one assembly where I bumped into this, that eggnog didn't return any results. We'll see

Reply:
Hmm, interesting. I would find that very unusual, that might have been an eggNOG issue of some sort. I will leave it as required and if we bump into this again, will investigate.